Code/GraphMol/Depictor/Templates.h

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright (C) 2003-2022 Greg Landrum and other RDKit contributors
```

Comment:
Copyright needs to be fixed

Reply:
I think I just copied and pasted from another file -- what should I have here? Do I just need to update the year?